hydrogym/ts.py

```diff
@@ -163,15 +224,15 @@ def step(self, iter, control=None):
         if control is not None:
             control = self.enlist_controls(control)
             self.update_controls(control)
-            for (B, ctrl) in zip(self.B, self.control):
+            for (B, ctrl) in zip(self.B, self.state):
                 Bu, _ = B.split()
                 self.u += Bu * ctrl
 
-        logging.log(logging.DEBUG, "Velocity predictor done, solving Poisson")
+        logging.log(logging.DEBUG, f"Velocity predictor done, solving Poisson")
```

Comment:
Do you have the pre-commit hooks Ludger added installed for formatting and linting?  If I remember correctly flake8 doesn't like the f-strings if there's no brackets in them (this was my fault originally but should be fixed now)

Reply:
oh crap, I thought we were doing black via #39 where Ludger says "Shall we go with black then? It does seem to be a pretty standard choice for most of the modern Python projects."

Reply:
Oh yeah, it is black but also with flake8 through the [pre-commit](https://pre-commit.com/) config he set up. Do you have that installed? It should take care of all this automatically anytime you commit.  This was new to me as well, so let me know if you have any trouble getting it set up.